Sergiu Bogos/readme.md

```diff
@@ -0,0 +1,28 @@
+- Inserarea unui student
+```
+INSERT INTO students(name, faculty_id, group_id) VALUES ('placeholder', 1, 1)
+```
+
+- Afișarea detaliilor despre un student
+```
+SELECT * FROM students WHERE id = 1
+```
+
+- Modificarea unui student
+```
+UPDATE students SET name = 'placeholder', faculty_id = 1, group_id = 1 WHERE id = 1
+```
+
+- Afișarea top 10 studenți cu cea mai bună notă medie
+```
+SELECT students.name, AVG(mark) as top FROM marks LEFT JOIN students on marks.student_id = students.id GROUP BY marks.student_id ORDER BY top DESC LIMIT 10 ;
+```
+
+- Top studenți per materie
+```
+¯\_(ツ)_/¯
```

Comment:
Ai putea sa selectezi AVG(mark) cu order by + grupezi dupa materie si student in acelasi moment :)